_posts/0000-01-02-sumesha22.md

```diff
@@ -0,0 +1,6 @@
+---
+layout :slide
+title : "welcome to our second slide !"
+---
+your text
```

Comment:
Global warming is one of the biggest threat that humans and all the living being in this world will be facing in the future,,